src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/caseupdate/CaseUpdateClient.java

```diff
@@ -57,10 +65,18 @@ public SuccessfulUpdateResponse updateCase(
             caseUpdate.exceptionRecord.id
         );
 
-        return restTemplate.postForObject(
+        SuccessfulUpdateResponse response = restTemplate.postForObject(
             url,
             new HttpEntity<>(caseUpdate, headers),
             SuccessfulUpdateResponse.class
         );
+
+        Set<ConstraintViolation<SuccessfulUpdateResponse>> violations = validator.validate(response);
+
+        if (violations.isEmpty()) {
+            return response;
+        } else {
+            throw new ConstraintViolationException(violations);
+        }
```

Comment:
Suggestion: worth extracting some code out of this method - validation or call + validation.